.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
```

Comment:
The Build workflow will always run on PRs, main branch, and version tags. It will upload the build artifact that can be accessed by downstream workflows with 1 day retention (default is 30 days).